app/routes/_index.tsx

```diff
@@ -28,8 +29,8 @@ export const meta: V2_MetaFunction = () => {
 export default function Index() {
   const tekster = useTekster(ESanitySteg.FORSIDE);
   const { knappStart } = useTekster(ESanitySteg.FELLES);
+  const [erSamtykkeBekreftet, settErSamtykkeBekreftet] = useBekreftetSamtykke();
```

Comment:
Trenger vi sett-funksjonen her også? Ser ut til at verdien allerede blir satt i SamtykkePanel

Reply:
Virker som logikken til Samtykkepanel er både _et steg opp_ og i komponentet selv. Alternativt kan man proppe ned `erSamtykkeBekreftet` ned i Samtykke panel? Da slipper man å bruke den hooken der? Man bruker variabelen for å si om man skal vise feilmelding i komponentet. Tanker? @kristeine 

**Alternativer å vurdere:** 
1. Proppe ned `erSamtykkeBekreftet` ned i SamtykkePanel
2. Berre bruke context for å hent det direkte

Reply:
En nyttig samtale seinere førte til at vi satt at man kun ender samtykke på et sted i koden: 1e63cd7